lib/set.rb

```diff
@@ -168,6 +168,28 @@ def clear
     self
   end
 
+  # Removes all nil elements from self. Returns self if any elements removed,
```

Comment:
Should read "Removes the nil element from self. Returns self if the element was removed"

Reply:
Sure! 💦 I fixed the doc.
Note: It seems that there is almost no marking using `+` in this code, so I omitted it now. I will add it to this if needed.